### DISCUSSION TITLE
413 "Request Entity Too Large" error when trying to upload files

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#5332] [Error / Bug Report] [answered]
Hi,

I'm trying to upload a file which is about 1.5 MB and I get a 413 "Request Entity Too Large" error.
Tried different files with different sizes, also tried uploading into different folders.
Most likely there's a 1MB limit, even tho wiki says that it accepts files up to 5MB.
Is there some config limit I'm not aware about?

**Update:**

* changing file limit under Security tab does not affect anything, also, this text in upload window does not get updated
![image](https://user-images.githubusercontent.com/5992169/170256588-b66d8d13-89f1-4406-9743-7f124f442f69.png)
* I found this open issue, tried suggestions in it, but nothing worked
https://github.com/Requarks/wiki-v1/issues/49

Will this be fixed? This bug is stopping our migration process :(

**Comment by ghost** [comment, marked as answer]
We found out the problem:
apparently njinx has a 1MB file upload limit
https://docs.rackspace.com/support/how-to/limit-file-upload-size-in-nginx/
![image](https://user-images.githubusercontent.com/5992169/170257478-49a3fd6e-0903-4d0f-984d-2bca0d3f8bd6.png)

Seems like wiki.js limits should overwrite njinx, because now two limits have to be set separately?

**Reply by NGPixel** [reply]
That's not how reverse proxying works. You can't just "override" a system that processes a request first.

You need to set both, there's no workaround to this.

**Reply by ghost** [reply]
Thank you.
